Rakefile

```diff
@@ -2,7 +2,27 @@
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
 
 require_relative 'config/application'
+require 'solr_wrapper/rake_task' unless Rails.env.production?
+require 'rspec/core/rake_task'
 
 Rails.application.load_tasks
 
-require 'solr_wrapper/rake_task' unless Rails.env.production?
+Rake::Task["test"].clear
+
+desc 'Execute the test build that runs on travis'
+task :test => [:environment] do
+  require 'solr_wrapper'
+  if Rails.env.test?
+    Rake::Task['db:migrate'].invoke
+    SolrWrapper.wrap do |solr|
+      solr.with_collection(name: 'blacklight-core', dir: "#{Rails.root}/solr/conf") do
+        Rake::Task['geoblacklight:solr:seed'].invoke
+        Rake::Task['spec'].invoke
+      end
+    end
+  else
+    system('rake ci RAILS_ENV=test')
```

Comment:
this block is unnecessary imo, as we don't have a `rake ci` task to call, and `Rails.env.test?` is always `true`

Reply:
i think we shuold change this to `rake test`.  But yeah this is necessary as some testing gems are not available in other environments

Reply:
ah ok. I'm good with `rake test` then